sdk/src/clock.rs

```diff
@@ -29,7 +29,8 @@ pub const NUM_CONSECUTIVE_LEADER_SLOTS: u64 = 4;
 pub const MAX_HASH_AGE_IN_SECONDS: usize = 120;
 
 // This must be <= MAX_HASH_AGE_IN_SECONDS, otherwise there's risk for DuplicateSignature errors
-pub const MAX_RECENT_BLOCKHASHES: usize = MAX_HASH_AGE_IN_SECONDS;
+pub const MAX_RECENT_BLOCKHASHES: usize =
```

Comment:
Hmm the above comment worries me. MAX_RECENT_BLOCKHASHES <= MAX_HASH_AGE_IN_SECONDS is not enforced anymore with this change

Reply:
we can add a compile time assert, https://doc.rust-lang.org/1.5.0/std/macro.assert!.html

Reply:
After double checking, it's fine. Originally this was the issue: https://github.com/solana-labs/solana/pull/3679,  

Since that issue, status cache has been changed to no longer use MAX_HASH_AGE_IN_SECONDS, https://github.com/solana-labs/solana/blob/master/runtime/src/status_cache.rs#L10, so it is all good

Reply:
Maybe it's worth keeping the other constant around. It gives some meaning to this value. I have no strong opinion though.

Reply:
ok you win